README.md

```diff
@@ -144,4 +144,4 @@ If you'd like to learn more about them, consider these resources:
 
 ## License
 
-[MIT License](LICENSE)
+[MIT License](LICENSE)
```

Comment:
ummm actually this is added for linux support because the last line of a file may be skipped due to lack of newline

Reply:
noooooooooooooooo

Reply:
ok nerd shut up